nltk/tag/crfsuite.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Interface to the CRFSuite Tagger
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+A module for interfacing with the CRFSuite taggers.
```

Comment:
Currently it is a module for POS tagging using CRFsuite. 

There is no explicit documentation about how to use your own features (and `_get_features` method is private), and there is no way to set trainer parameters - regularization and other constants, training algorithms. If one wants to develop e.g. a NER system using this tagger then all of these is helpful: custom features are must, default constants should be tuned, and it is good to have more information about the training process (verbose=True parameter, usage of the holdout dataset, etc).